rma_reason_code/views/rma_order_line_views.xml

```diff
@@ -48,7 +48,7 @@
                     widget="many2many_tags"
                     options="{'color_field': 'color', 'no_create': True}"
                 />
-                <field name="allowed_reason_code_ids" invisible="1" />
+                <field name="allowed_reason_code_ids" column_invisible="1" />
```

Comment:
This is a form view, column_invisible only applies to tree/list views

![image](https://github.com/user-attachments/assets/457e0265-b70b-4403-9e4f-28d9523518b7)

Reply:
done x2!